modules/Bio/EnsEMBL/DBSQL/TranscriptSupportingFeatureAdaptor.pm

```diff
@@ -63,26 +63,34 @@ use vars qw(@ISA);
 
   Arg [1]    : Bio::EnsEMBL::Transcript $transcript 
                The transcript to fetch supporting features for
+  Arg [2]	 : String (optional)
```

Comment:
Your editor still seems to be adding tabs, so things aren't aligning properly, please check your editor so these are spaces.